lib/tty/pager/system.rb

```diff
@@ -12,32 +12,23 @@ class SystemPager < Pager
       # @example Basic usage
       #   available # => 'less'
       #
-      # @example Usage with commands
-      #   available('less', 'cat')  # => 'less'
-      #
-      # @param [Array[String]] commands
-      #
       # @return [String]
       #
       # @api public
-      def self.available(*commands)
-        commands = commands.empty? ? executables : commands
-        commands.compact.uniq.find { |cmd| command_exists?(cmd) }
+      def self.available
+        executables.compact.uniq.find { |cmd| command_exists?(cmd) }
```

Comment:
I kept this as-is since that's what it was beforehand, but I'm happy to change as well.